tests/files/test-api-client.php

```diff
@@ -16,7 +16,11 @@ public static function setUpBeforeClass() {
 	public function setUp() {
 		parent::setUp();
 
-		$this->api_client = new API_Client( 'https://files.go-vip.co', 123456, 'super-sekret-token' );
+		$this->api_client = new API_Client(
+			'https://files.go-vip.co',
+			123456,
+			'super-sekret-token',
+			API_Cache::get_instance() );
```

Comment:
Thoughts on passing in a mock/stub version of `API_Cache` instead? I guess with the current approach it's nice that we end up testing the integration between `API_Client` and `API_Cache`

Reply:
I'm already working on using a mock/stub cache for those test